config/MezaCoreExtensions.yml

```diff
@@ -294,11 +294,10 @@ list:
 #    version: "{{ mediawiki_default_branch }}"
 #    legacy_load: true
 
-#  # SemanticDrilldown - Provides a page for "drilling down" through a site's data, using categories and filters on semantic properties
-#  - name: SemanticDrilldown
-#    repo: https://github.com/SemanticMediaWiki/SemanticDrilldown.git
-#    version: master
-#    legacy_load: true
+  # SemanticDrilldown - Provides a page for "drilling down" through a site's data, using categories and filters on semantic properties
+  - name: SemanticDrilldown
+    repo: https://github.com/freephile/SemanticDrilldown.git
+    version: master
```

Comment:
SemanticDrilldown **broke** deploys due to a bug, and was removed. We fixed the bug in the freephile forked repo and re-enabled the extension by pointing to our fixed copy.